exercises/practice/largest-series-product/src/test/java/LargestSeriesProductCalculatorTest.java

```diff
@@ -155,7 +155,7 @@ public void testNegativeSeriesLengthIsRejected() {
 
         assertThatExceptionOfType(IllegalArgumentException.class)
                 .isThrownBy(() -> calculator.calculateLargestProductForSeriesLength(-1))
-                .withMessage("Series length must be non-negative.");
+                .withMessage("Series length must not be negative.");
```

Comment:
As mentioned in another comment, let's keep the message as it was.

Reply:
ok I have done a rollback for this